tests/unit/single_table/test_base.py

```diff
@@ -0,0 +1,63 @@
+from unittest.mock import Mock, patch
+
+from sdv.single_table.base import BaseSynthesizer
+
+
+class TestBaseSynthesizer:
+
+    @patch('sdv.single_table.base.DataProcessor')
+    def test___init__(self, mock_data_processor):
+        """Test instantiating with default values."""
+        # Setup
+        metadata = Mock()
+
+        # Run
+        instance = BaseSynthesizer(metadata)
+
+        # Assert
+        assert instance.enforce_min_max_values
```

Comment:
I'd assert that it `is True`, just to make sure no other values are being passed. Same in the line below.